src/components/step-sequence/step-sequence-item/step-sequence-item.stories.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { text, select } from '@storybook/addon-knobs';
+import StepSequenceItem from './step-sequence-item';
+import OptionsHelper from '../../../utils/helpers/options-helper/options-helper';
+
+storiesOf('Step Sequence Item', module).add('default', () => {
+  const indicator = text('indicator', '1');
+  const status = select('status', OptionsHelper.steps, OptionsHelper.steps[0]);
```

Comment:
Can we take the default value from the component? I.e. `StepSequenceItem.defaultProps.status`